xlineapi/src/lib.rs

```diff
@@ -781,4 +781,82 @@ mod test {
 
         assert!(!read_write_nested_txn_req.is_read_only());
     }
+
+    #[test]
+    fn txn_request_is_serializable_should_success() {
+        let serializable_req = Some(Request::RequestRange(RangeRequest {
+            serializable: true,
+            ..Default::default()
+        }));
+
+        let serializable_txn_req = TxnRequest {
+            compare: vec![],
+            success: vec![RequestOp {
+                request: serializable_req.clone(),
+            }],
+            failure: vec![RequestOp {
+                request: serializable_req.clone(),
+            }],
+        };
+
+        assert!(serializable_txn_req.is_serializable());
```

Comment:
We should cover the happy path and the sad path here. Given that `is_serializable` is a recursive implementation, we'd better create a dedicated unit test in order to cover all possible branches.